core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowAkkaPlugin.scala

```diff
@@ -90,7 +90,8 @@ object CloudflowAkkaPlugin extends AutoPlugin {
         Instructions.User(UserInImage),
         Instructions.EntryPoint.exec(Seq("bash", "/opt/akka-entrypoint.sh")),
         Instructions.Copy(sources = Seq(CopyFile(depJarsDir)), destination = OptAppDir, chown = Some(userAsOwner(UserInImage))),
-        Instructions.Copy(sources = Seq(CopyFile(appJarsDir)), destination = OptAppDir, chown = Some(userAsOwner(UserInImage)))
+        Instructions.Copy(sources = Seq(CopyFile(appJarsDir)), destination = OptAppDir, chown = Some(userAsOwner(UserInImage))),
+        Instructions.Expose(Seq(5005))
```

Comment:
Thinking twice about it, I think we should make this feature optional.
For security reasons, I can easily see customers that do want to not expose ports in production environments.

Reply:
Can we put this in as an optional sbt setting?